parcels/particleset.py

```diff
@@ -158,17 +161,20 @@ def from_line(cls, fieldset, pclass, start, finish, size, depth=None, time=None,
         :param depth: Optional list of initial depth values for particles. Default is 0m
         :param time: Optional start time value for particles. Default is fieldset.U.time[0]
         :param repeatdt: Optional interval (in seconds) on which to repeat the release of the ParticleSet
+        :param lonlatdepth_dtype: Floating precision for lon, lat, depth particle coordinates.
+               It is either np.float32 or np.float64. Default is np.float32 if fieldset.U.interp_method is 'linear'
+               and np.float64 if the interpolation method is 'cgrid_velocity'
         """
 
         lonlat_type = cls.lonlatdepth_dtype_from_field_interp_method(fieldset.U)
```

Comment:
if `lonlatdepth_type` is defined, it should overwrite the default value of `lonlat_type`